client/src/modules/privacy-policy/components/PrivacyPolicy.en.md

```diff
@@ -0,0 +1,233 @@
+
+### [7. IS YOUR INFORMATION TRANSFERRED INTERNATIONALLY?](#7-is-your-information-transferred-internationally)
+
+**_In Short:_** We may transfer, store, and process your information in countries other than your own.
+
+Our servers are located in Poland. If you are accessing our Sites or Apps from outside Poland, please be aware that your information may be transferred to, stored, and processed by us in our facilities and by those third parties with whom we may share your personal information (see ["WILL YOUR INFORMATION BE SHARED WITH ANYONE?"](#3-will-your-information-be-shared-with-anyone) above), in United States, and other countries.
+
+If you are a resident in the European Economic Area, then these countries may not have data protection or other laws as comprehensive as those in your country. We will however take all necessary measures to protect your personal information in accordance with this privacy policy and applicable law.
+
+### [8. HOW LONG DO WE KEEP YOUR INFORMATION?](#8-how-long-do-we-keep-your-information)
+
+**_In Short:_** We keep your information for as long as necessary to fulfill the purposes outlined in this privacy policy unless otherwise required by law.
+
+We will only keep your personal information for as long as it is necessary for the purposes set out in this privacy policy, unless a longer retention period is required or permitted by law (such as tax, accounting or other legal requirements). No purpose in this policy will require us keeping your personal information for longer than the period of time in which users have an account with us.
+
+When we have no ongoing legitimate business need to process your personal information, we will either delete or anonymize it, or, if this is not possible (for example, because your personal information has been stored in backup archives), then we will securely store your personal information and isolate it from any further processing until deletion is possible.
+
+### [9. HOW DO WE KEEP YOUR INFORMATION SAFE?](#9-how-do-we-keep-your-information-safe)
+
+**_In Short:_** We aim to protect your personal information through a system of organisational and technical security measures.
+
+We have implemented appropriate technical and organisational security measures designed to protect the security of any personal information we process. However, please also remember that we cannot guarantee that the internet itself is 100% secure. Although we will do our best to protect your personal information, transmission of personal information to and from our Sites or Apps is at your own risk. You should only access the services within a secure environment.
```

Comment:
`Although we will do our best to protect your personal information, the transmission of personal information to and from our Sites or Apps is at your own risk`   
--> the transmission